website/docs/d/oracle_connection.html.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: "pureport"
+page_title: "Pureport: pureport_oracle_connection"
+sidebar_current: "docs-pureport-datasource-oracle_connection"
+description: |-
+  Manages a Pureport Oracle Fast Connect Connection.
+---
+
+# Data Source: pureport\_oracle\_connection
+
+## Example Usage
+
+```hcl
+data "pureport_oracle_connection" "basic" {
+  connection_id = data.pureport_connections.main.connections.0.id
+
+  filter {
+    name = "Name"
+    values = ["^My Conn*"]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `connection_id` - (Required) The ID of the connection. You should use the `pureport_connections` data source
+  for querying the list of available connections and discover the ID for the connection.
+
+- - -
+
+* `filter` - (Optional) A filter used to scope the list e.g. by tags.
+  * `name` - (Required) The name of the filter. The valid values are defined in the
+    [Pureport SDK Model](https://github.com/pureport/pureport-sdk-go/blob/master/docs/client/AwsDirectConnectConnection.md).
+    Nested values are supported. E.g.("Location.DisplayName")
+  * `values` - (Required) The value of the filter. Currently only regex strings are supported.
+
+## Attributes
+
+* `name` - The name for the connection
+* `location_href` - HREF for the Pureport Location to attach the connection.
+* `network_href` - HREF for the network to associate the connection.
```

Comment:
maybe nit picking but this description seems awkward to me.
`HREF for the network the connection belongs to` maybe? or something similar?

Reply:
Updated.